### ISSUE TITLE
How to determine the number of tracks normalized at a time

### ISSUE THREAD

**Issue by 12num** [InhwanBae/EigenTrajectory#3]
Thanks for your work

You propose to use SVD to construct ET space for normalized trajectory A(L*N). I'm not quite sure if N here represents the number of all tracks in each training set ("eth" "hotel" "univ" "zara1" "zara2")?

**Comment by InhwanBae** [comment]
Hi @12num,
Exactly. Before starting the training, all trajectories within the training dataset are decomposed using SVD to construct ET space. It will be much easier to understand if you look at the code below.

https://github.com/InhwanBae/EigenTrajectory/blob/a2596fca73e5c30188c1572930b1bb773aa04167/utils/trainer.py#L48-L54

**Comment by 12num** [comment]
I see, thank you very much!

**Comment by InhwanBae** [comment]
I'm closing this issue for now. Feel free to open another issue if you have any further questions!
